### ISSUE TITLE
2.9.4 Set color/category to will not fill in the rectangle color until clicking on dropdown 

### ISSUE THREAD

**Issue by fogoat** [phw198/OutlookGoogleCalendarSync#1436]
> Checklist:-
> * [ ] [Upgrade](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Upgrading) to the latest release (alpha if available) ![latest release](https://img.shields.io/github/release/phw198/OutlookGoogleCalendarSync/all.svg)
> * [ ] Reproduce the issue
> * [ ] Complete the information below
> * [ ] Provide your `OGcalsync.log` file (see [how-to](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Reporting-problems#locating-your-ogcalsynclog-file))

**OGCS Version**: v2.9.4.0
**Edition**: Portable  
**Sync Direction**: n/a? / O->G 

**Bug Description**  

I also notice that set colour/category to Calendar Default will not display the filled in color (it is blank/white) until I click on the dropdown list displaying the various colors.

**Comment by phw198** [comment]
That's a "feature" not a bug :wink: 

Basically, when OGCS first loads it has not communicated with Google and therefore it doesn't know what the colour for calendar is - not until you click on the dropdown and then that will trigger a connection (which is why there is a delay the first time you do this). If you've already done a sync or retrieve the calendars (ie connected to Google), then the colour will show when you view the settings.
